module/platform/Webots/data/config/camera.yaml

```diff
@@ -0,0 +1,39 @@
+# serial_number: 0
+
+lens:
+  # Lens:
```

Comment:
What are all these comments for

Reply:
Will be populated when i know what they are